lib/JSON/Validator.pm

```diff
@@ -250,9 +250,9 @@ sub _load_schema {
   if (-e $file) {
     $file = $file->realpath;
     warn "[JSON::Validator] Loading schema from file: $file\n" if DEBUG;
-    return $self->_load_schema_from_text(\$file->slurp), CASE_TOLERANT ? lc $file : $file;
+    return $self->_load_schema_from_text(\$file->slurp), CASE_TOLERANT ? path(lc $file) : $file;
```

Comment:
Why path on `lc $file` but not on `$file`?

Reply:
2 lines above, it's visibly a `Mojo::File`. If you `lc` that, it stringifies it, and `path` then turns it back to a `Mojo::File`. It needs to be that, because if it's a string, it gets made into a `Mojo::URL`, which causes problems.

Reply:
Ah! Thank you 👍